cumulusci.yml

```diff
@@ -77,11 +77,17 @@ tasks:
             path: unpackaged/config/trial
             namespace_tokenize: $project_config.project__package__namespace
 
-    add_picklist_values:
+    add_application_type_picklist_values:
         description: Adds picklist values to the given field, if they don't already exist.
         class_path: tasks.add_picklist_values.AddPicklistValues
         group: 'K-12: Metadata'
-        
+        options:
+            sobject: hed__Application__c
+            field: hed__Application_Type__c
+            values: "Elementary,Middle School,High School"
```

Comment:
Interesting! I realize now there might be a missing gap in this custom CCI task that I built. We built it with the intention of adding picklist values, but I suppose there might also be a need to remove picklist values?

@lifewithryan thoughts on this? What are the plans for Applications in K-12 moving forward? (Do the existing values of Undergraduate, Graduate, etc for Application Type make sense?)

Reply:
@robertbwatson  Can we pair on that delete_picklist_values task?

Reply:
I am not sure we ever want to "delete" picklist values. There's always the chance that if they went out, someone is using them. I suppose if we can guarantee it is a new install (ala something being done via a trial org) we could do it, but definitely not as part of the installer.